### PR TITLE
update ort_run_frozen script

### PR DESCRIPTION
**Pull request by xysmlx**
* more data_types in ort_run_frozen script
* more running options in ort_run_frozen script: ONNX model format check, GraphOptimizationLevel, logger_severity

### CODE REVIEW

**Review by wenxcs — DISMISSED** [review]

File: models/pytorch2onnx/ort_run_frozen.py [hunk 1]
```diff
@@ -29,6 +34,10 @@ def get_numpy_dtype(onnx_dtype):
             return np.float32
```
Comment (mzmssg) [inline models/pytorch2onnx/ort_run_frozen.py:34]: remove `float32` & `float64`? Float type in ONNX only includes `float16`, `float`, `double`. Ref: https://github.com/onnx/onnx/blob/63a586914d51fdfd6d8668e81de043e3d9b0b382/onnx/defs/data_type_utils.h#L28

**Review by mzmssg — DISMISSED** [review]
Besides, do you plan update the benchmark with ort IO binding in this PR?

File: models/pytorch2onnx/ort_run_frozen.py (hunk 1 above)
Comment (xysmlx) [reply models/pytorch2onnx/ort_run_frozen.py]: sure, done

**Review by mzmssg — APPROVED** [review]

**Comment by xysmlx** [comment]
> Besides, do you plan update the benchmark with ort IO binding in this PR?

I think the end-to-end performance test should include H2D and D2H.

**Pull request merged**